src/main/scala/scalismo/common/DiscreteField.scala

```diff
@@ -117,11 +117,11 @@ object DiscreteVectorField {
    * the vector is ordered as (v_11, v_12, ... v_1d, ...v_n1, v_n2, v_nd)
    */
   def fromDenseVector[D <: Dim: NDSpace, DO <: Dim: NDSpace](domain: DiscreteDomain[D],
-    vec: DenseVector[Float]): DiscreteVectorField[D, DO] = {
+    vec: DenseVector[Float])(implicit vectorBuilder: Vector.Create[DO]): DiscreteVectorField[D, DO] = {
```

Comment:
Is there a reason we cannot create the vector using NDSpace, but need to pass the Vector.Create?